pom.xml

```diff
@@ -498,6 +498,17 @@
     <version.org.jvnet.jaxb2.maven2.maven-jaxb2-plugin>0.14.0</version.org.jvnet.jaxb2.maven2.maven-jaxb2-plugin>
     <version.org.mock-server>5.11.1</version.org.mock-server>
     <version.org.apache.kafka>2.6.0</version.org.apache.kafka>
+
+    <!-- ************** -->
+    <!-- Build settings -->
+    <!-- ************** -->
+
+    <version.shade.plugin>3.2.4</version.shade.plugin>
+
+    <!-- maven-compiler-plugin -->
+    <maven.compiler.target>11</maven.compiler.target>
+    <maven.compiler.source>11</maven.compiler.source>
+    <maven.compiler.release>11</maven.compiler.release>
```

Comment:
we need still to keep target for java 8, so this is not acceptable to be merged yet ;)

Reply:
Done.  Sorry about taking so long.

Reply:
@mareknovotny Removed.